config.json

```diff
@@ -488,6 +488,14 @@
         ],
         "difficulty": 2
       },
+      {
+        "slug": "series",
+        "name": "Series",
+        "uuid": "824cefbf-a25d-445c-abc4-d92824006de1",
+        "difficulty": 2,
+        "practices": [],
+        "prerequisites": []
```

Comment:
Both our solutions seem to

```json
"practices":     ["conditionals", "numbers", "strings", "compare", "variables", "functions"],
"prerequisites": ["conditionals", "numbers", "strings", "compare"],
```

It is possible to solve this exercise without defining any functions or variables.

Reply:
Whoops. I forgot to fill those out. Added. Thanks!